networkx/algorithms/chordal.py

```diff
@@ -344,3 +347,78 @@ def _connected_chordal_graph_cliques(G):
                 raise nx.NetworkXError("Input graph is not chordal.")
         cliques.add(frozenset(clique_wanna_be))
         return cliques
+
+
+@not_implemented_for('directed')
+def complete_to_chordal_graph(G):
+    """Complete to chordal graph
+
+    Enhances the graph by edges to receive a chordal graph. A graph
+    G=(V,E) is called chordal if for each cycle with length bigger
+    than 3, two non adjacent nodes are connected by an edge (called chord).
+
+    Parameters
+    ----------
+    G : NetworkX graph
+        Undirected graph
+
+    Returns
+    -------
+    H : NetworkX graph
+        The chordal enhancement of G
+    alpha : Dictionary
+            The elimination ordering of nodes of G
+
+    Notes
+    ------
+    There are different approaches to calculate the chordal
+    enhancement of a graph. The algorithm used here is called
+    MCS-M and gives at least minimal (local) triangulation of graph. Note
+    that this triangulation is not necessarily minimum minimum (global).
+
+    https://en.wikipedia.org/wiki/Chordal_graph
+
+    References
+    ----------
+    .. [1] Berry, Anne & Blair, Jean & Heggernes, Pinar & Peyton, Barry. (2004).
+           Maximum Cardinality Search for Computing Minimal Triangulations of Graphs.
+           Algorithmica. 39. 287-298. 10.1007/s00453-004-1084-3.
+
+    Examples
+    --------
+    >>> import networkx as nx
+    >>> from networkx.algorithms.chordal import complete_to_chordal_graph
+    >>> G = nx.wheel_graph(10)
+    >>> H,alpha = complete_to_chordal_graph(G)
+    """
+    if G is None:
+        raise ValueError("Expected NetworkX graph!")
+    H = G.copy()
+    alpha = {node: 0 for node in H.nodes()}
+    if nx.is_chordal(H):
+        return H, alpha
+    chords = []
+    weight = {node: 0 for node in H.nodes()}
+    unnumbered_nodes = list(H.nodes())
+    for i in range(len(H.nodes()), 0, -1):
+        # get the node in unnumbered_nodes with the maximum weight
+        z = [node for node in unnumbered_nodes
+             if weight.get(node) == max([weight.get(i)
+                                         for i in unnumbered_nodes])][0]
```

Comment:
To get the node with max weight use:

    z = max(unnumbered_nodes, key=lambda node: weight[node])

Reply:
You are right. This is better to understand.